db/seeds.rb

```diff
@@ -5,3 +5,7 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+# StreetCp.create(street_name: "Calle", cp: "28000")
+# Branch.create(name: "Calle", street_cp_id: 1, city: "Colima", town: "Villa De Alvarez", number: 955555555)
+# Employee.create(name: "Juan", email: "juan@gmail.com", position: "Gerente", private_number: 123456789, branch_id: 1)
+# Admin.create(email: "admin@admin.com", encrypted_password: "admin123")
```

Comment:
If you are not going to occupy this section of code, instead of commenting it out, it should be removed, to keep your code clean.

Reply:
thanks, done :100: